runBeforePushing.sh

```diff
@@ -0,0 +1,2 @@
+sbt clean test IntegrationTest/test scalafmtAll
```

Comment:
Did you mean to add this file?

Reply:
Certainly. It's referenced in the README, as well.

Reply:
@tna-erasmos can you also please add `IntegrationTest /scalafmt` to the fmt alias in `.sbtrc`

Reply:
Will do.

Reply:
I upgraded the `scalafmt` dep from `2.4.3` to `2.4.4`, as this will automatically include the integration tests, for both `scalafmtAll` and `scalafmtCheckAll`.

I also simplified the sbt aliases.